internal/integration/mysql_test.go

```diff
@@ -186,7 +186,7 @@ func TestMySQL_AddColumns(t *testing.T) {
 			&schema.Column{Name: "b", Type: &schema.ColumnType{Raw: "mediumblob", Type: &schema.BinaryType{T: "mediumblob"}}},
 			&schema.Column{Name: "c", Type: &schema.ColumnType{Raw: "blob", Type: &schema.BinaryType{T: "blob"}}},
 			&schema.Column{Name: "d", Type: &schema.ColumnType{Raw: "longblob", Type: &schema.BinaryType{T: "longblob"}}},
-			&schema.Column{Name: "e", Type: &schema.ColumnType{Raw: "binary", Type: &schema.BinaryType{T: "binary"}}},
+			&schema.Column{Name: "e", Type: &schema.ColumnType{Raw: "binary(255)", Type: &schema.BinaryType{T: "binary(255)"}}},
```

Comment:
Please add a case instead of deleting an existing one

Reply:
I am not sure why it not pass without a size.  

If I don't give a size it will provide a default non-zero size for binary ?

Reply:
If an existing test fails because of a change that was made, it generally should mean that the change is not backward compatible (i.e it changes the existing behavior of the system).

Reply:
Just checked it with both MySQL 8 and 5.7 and that's the expected behavior.

#### Version 8
<img width="629" alt="image" src="https://user-images.githubusercontent.com/7413593/174468184-468a7ae4-eedb-4df3-9b0f-7eacdebeaee1.png">

#### Version 5.7

<img width="597" alt="image" src="https://user-images.githubusercontent.com/7413593/174468218-e3979131-64b2-412e-bea4-0cb5f58996de.png">

Reply:
👍